WindowsServerDocs/identity/ad-fs/operations/AD-FS-support-for-alternate-hostname-binding-for-certificate-authentication.md

```diff
@@ -1,38 +1,44 @@
 ---
-description: "Learn more about: AD FS support for alternate hostname binding for certificate authentication"
-ms.assetid: 4b71b212-7e5b-4fad-81ee-75b3d1f27869
 title: AD FS support for alternate hostname binding for certificate authentication
+description: Learn more about: AD FS support for alternate hostname binding for certificate authentication.
+ms.topic: article
 author: billmath
 ms.author: billmath
-manager: femila
-ms.date: 05/31/2017
-ms.topic: article
+ms.date: 01/03/2023
+
 ---
 # AD FS support for alternate hostname binding for certificate authentication
 
-On many networks the local firewall policies might not allow traffic through non-standard ports like 49443. This became an issue when trying to accomplish certificate authentication with AD FS prior to AD FS in Windows Server 2016. This is because you could not have different bindings for device authentication and user certificate authentication on the same host. The default port 443 is bound to receive device certificates and cannot be altered to support multiple binding in the same channel. The results were that smart card authentication would not work and users were unaware of what happened since there is no indication of what really happened.
+Applies to: Windows Server 2016 and later
+
+On many networks the local firewall policies may not allow traffic through non-standard ports like 49443. This was an issue when trying to accomplish certificate authentication with AD FS prior to Windows Server 2016 because different bindings for device authentication and user certificate authentication on the same host is not possible in earlier versions. Prior to Windows Server 2016, default port 443 is bound to receive device certificates and cannot be altered to support multiple binding in the same channel. Smart card authentication does not work and there is no notification to users explaining the cause.
```

Comment:
```suggestion
On many networks, the local firewall policies may not allow traffic through non-standard ports like 49443. Non-standard ports created issue when trying to accomplish certificate authentication with AD FS prior to Windows Server 2016 because different bindings for device authentication and user certificate authentication on the same host isn't possible. Prior to Windows Server 2016, default port 443 is bound to receive device certificates and can't be altered to support multiple binding in the same channel. Smart card authentication doesn't work and there's no notification to users explaining the cause.
```